jobs/quicksuggest2bq/tests/test_main.py

```diff
@@ -19,6 +19,7 @@
         "sample da",
         "sample dat",
     ],
+    "full_keywords": [("sample data", 3)],
```

Comment:
Would you kindly add one more full keyword so that we can test the case of "multiple full keywords" ?